tests/providers/google/cloud/operators/test_gcs_system.py

```diff
@@ -22,6 +22,7 @@
 from tests.test_utils.gcp_system_helpers import CLOUD_DAG_FOLDER, GoogleSystemTest, provide_gcp_context
 
 
+@pytest.mark.system("google.cloud")
```

Comment:
This test have already system market. See: https://github.com/apache/airflow/blob/d4c4db8a1833d07b1c03e4c062acea49c79bf5d6/tests/test_utils/gcp_system_helpers.py#L121

Reply:
Missed that - thanks, reverted